engines/python/setup/djl_python/huggingface.py

```diff
@@ -96,6 +96,9 @@ def get_rolling_batch_class_from_str(rolling_batch_type: str, is_mpi: bool,
     elif rolling_batch_type == "lmi-dist":
         from djl_python.rolling_batch.lmi_dist_rolling_batch import LmiDistRollingBatch
         return LmiDistRollingBatch
+    elif rolling_batch_type == "lmi-dist-v2":
```

Comment:
why not just replace lmi-dist?

Reply:
this will be done soon once the wheel is built and added to the container.